README.md

```diff
@@ -103,15 +119,34 @@ by adding -DUSE_CPU_ONLY=1 to the cmake command line.
     # Install build depedencies
     sudo apt install libboost-dev libboost-program-options-dev libboost-filesystem-dev opencl-headers ocl-icd-libopencl1 ocl-icd-opencl-dev zlib1g-dev
 
-    # Use stand alone directory to keep source dir clean
+    # Use a stand alone build directory to keep source dir clean
     mkdir build && cd build
+    
+    # Compile leelaz and autogtp in build subdirectory with cmake
     cmake ..
     cmake --build .
+    
+    # Optionnal : if you want to test if your build has no issue
```

Comment:
s/optionnal/optional

Reply:
i dont know what you mean here, sorry

Reply:
The issue is that optional is spelt with only one n